opensl_stream.c

```diff
@@ -465,7 +465,7 @@ int opensl_start(OPENSL_STREAM *p) {
     }
   }
   if (p->recorderRecord) {
-    memset(p->inputBuffer, 0, sizeof(p->inputBuffer));
+    memset(p->inputBuffer, 0, (size_t) (p->inputBufferFrames * p->inputChannels));
```

Comment:
What's wrong with the original version? And don't we have to multiply by `sizeof(short)`?